src/basic/btrfs-util.c

```diff
@@ -45,7 +46,6 @@
 #include "fileio.h"
 #include "io-util.h"
 #include "macro.h"
-#include "missing.h"
```

Comment:
we order includes alphabetically, see CODING_STYLE. If btrfs-ctree.h needs something from missing.h it should include it.